src/core/server.cc

```diff
@@ -47,6 +47,10 @@
 #include "src/core/pinned_memory_manager.h"
 #include "src/core/server.h"
 
+#include "src/backends/backend/triton_backend_manager.h"
+
```

Comment:
Remove extra lines after and before triton_backend_manager.h . 
Also if you check 
`#include "src/core/server.h"`
Is mistakenly added twice. Remove second occurrence.